src/browser/stores/dota.js

```diff
@@ -109,6 +112,38 @@ const DotaStore = Fluxxor.createStore({
         const leagueIds = this.patchIdsToLeagueIds[patchId];
 
         return leagueIds.map(id => this.leagueIdsToLeagueNames[id]);
+    },
+
+    getWinrate(combination) {   
+        let winCounter = 0;
+        let matchCounter = 0;
```

Comment:
instead of keeping matchCounter, you can just do `match.length` at the end.  saves some reading :)